admin/controllers/administrator.js

```diff
@@ -183,13 +189,44 @@ const getCheckWindows = async (req, res, next) => {
         id: cw._id,
         checkWindowName: cw.checkWindowName,
         adminStartDate: adminStartDateMo.format('DD MMM YYYY'),
-        checkDates: formatDate(checkStartDateMo, checkEndDateMo)
+        checkDates: formatDate(checkStartDateMo, checkEndDateMo),
+        canRemove: (Date.parse(cw.checkStartDate) >= Date.now()),
+        isCurrent: true
       }
     })
   } catch (error) {
     return next(error)
   }
 
+  // Get past check windows
+  try {
+    checkWindows = await checkWindowDataService.fetchPastCheckWindows(sortField, sortDirection)
+    checkWindowsPast = checkWindows.map(cw => {
+      const adminStartDateMo = moment(cw.adminStartDate)
+      const checkStartDateMo = moment(cw.checkStartDate)
+      const checkEndDateMo = moment(cw.checkEndDate)
+
+      return {
+        id: cw._id,
+        checkWindowName: cw.checkWindowName,
+        adminStartDate: adminStartDateMo.format('DD MMM YYYY'),
```

Comment:
There is a DateService that offers a format very similar to this, except that the format used by DateService is GDS compliant.

Reply:
'DD MMM YYYY' is what UX wants...